docs/workflow/advanced-usage/uploading-activation-maps.md

```diff
@@ -121,10 +121,10 @@ def create_and_upload_bitmap(
 
 !!! info inline end
     If you are not familiar with the workflow concept, please read the
-    [:kolena-cube-20: Building a Workflow](https://docs.kolena.io/building-a-workflow) guide.
+    [:kolena-cube-20: Building a Workflow](https://docs.kolena.io/workflow/building-a-workflow) guide.
```

Comment:
From anywhere within the docs tree, we should use relative links — validity of relative links is checked during docs build, so these would have been caught prior to pushing.